_scripts/instruction-widget/templates/install/header.html

```diff
@@ -1 +1,13 @@
+<aside class="note">
+    <h4>Note:</h4>
+    <p>Certbot is meant to be run on the server where your website is hosted.
+If you don't have <a href="https://en.wikipedia.org/wiki/Superuser">
+administrative</a> <a href="https://en.wikipedia.org/wiki/Shell_account">
+shell access</a> to your webserver or you're not familiar with command line server
+administration, you should check if your hosting provider has built-in Let's
+Encrypt support by contacting them or <a
+href="https://community.letsencrypt.org/t/web-hosting-who-support-lets-encrypt/6920">
+checking this list of supporting providers</a>.
+</p>
+<p</aside>
```

Comment:
The `<p` actually causes the entire install section to be included in the note in my browser.